docs/content/latest/deploy/kubernetes/clients.md

```diff
@@ -109,3 +109,112 @@ Status:
         router: no errors
         loadbalancer emulator: no errors
 ```
+
+## Connecting TLS Secured YugabyteDB cluster deployed by Helm Charts
+
+To bring up a YugabyteDB cluster with encryption in transit (TLS) enabled, follow the [instructions here](https://docs.yugabyte.com/latest/deploy/kubernetes/single-zone/gke/helm-chart/) and set the flag `tls.enabled=true` in the helm command-line.
+
+For example, `helm install yugabyte --namespace yb-demo --name yb-demo --set=tls.enabled=true`.
+
+### Connecting from within the Kubernetes cluster
+
+Copy the following `yb-client.yaml` and use this `kubectl create -f yb-client.yaml` command to create a pod with auto-mounted client certificates.
+
+```yaml
+apiVersion: v1
+kind: Pod
+metadata:
+  name: yb-client
+  namespace: yb-demo
+spec:
+  containers:
+  - name: yb-client
+    image: yugabytedb/yugabyte-client:latest
+    env:
+    - name: SSL_CERTFILE
+      value: "/root/.yugabytedb/root.crt"
+    volumeMounts:
+    - name: yugabyte-tls-client-cert
+      mountPath: "/root/.yugabytedb/"
+  volumes:
+  - name: yugabyte-tls-client-cert
+    secret:
+      secretName: yugabyte-tls-client-cert
+      defaultMode: 256
+```
+
+Here is an example of a client that uses the `YSQL shell` ([`ysqlsh`](../../../admin/ysqlsh)) to connect.
+
+Use the following command to verify the connection.
+
+```sh
+$ kubectl exec -n yb-demo -it yb-client -- ysqlsh -h yb-tservers.yb-demo.svc.cluster.local "sslmode=require"
+ysqlsh (11.2-YB-2.1.5.0-b0)
+SSL connection (protocol: TLSv1.2, cipher: ECDHE-RSA-AES256-GCM-SHA384, bits: 256, compression: off)
+Type "help" for help.
+
+yugabyte=# \conninfo
+You are connected to database "yugabyte" as user "yugabyte" on host "yb-tservers.yb-demo.svc.cluster.local" at port "5433".
```

Comment:
```suggestion
You are connected to the `yugabyte` database as user `yugabyte` on host `yb-tservers.yb-demo.svc.cluster.local` at port `5433`.
```

Reply:
This can't be edited as this is the output of the `\conninfo` command.

Reply:
I missed that -- OK as-is then.